pipeline/inputs/exec.md

```diff
@@ -8,7 +8,7 @@ command arguments could lead to malicious command execution.
 
 ## Container support
 
-This plugin will not function in the distroless production images (AMD64 currently) as it needs a functional `/bin/sh` which is not present.
+This plugin will not function in the distroless production images (AMD64 and s390x currently) as it needs a functional `/bin/sh` which is not present.
```

Comment:
This needs updating, all production images are distroless so this was out of date for a while. No container images support this plugin other than the debug ones.

Reply:
updated..